core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJProvider.scala

```diff
@@ -140,10 +144,13 @@ final case class IntelliJFactory(
   }
 
   private def installIntelliJ(version: IntelliJVersion, root: Path): Unit = {
-    println(s"Installing $version")
```

Comment:
I'd like to keep this message. When probe starts it reports the progress this way (installing IJ, later installing plugins).

Reply:
removed